main.py

```diff
@@ -50,10 +51,13 @@ def main(POST_ID=None) -> None:
     redditid = id(reddit_object)
     length, number_of_comments = save_text_to_mp3(reddit_object)
     length = math.ceil(length)
+    bg_config = {}
     get_screenshots_of_reddit_posts(reddit_object, number_of_comments)
-    bg_config = get_background_config()
-    download_background(bg_config)
-    chop_background_video(bg_config, length, reddit_object)
+    bg_config["video"] = get_background_config("video")
```

Comment:
could simplify to bg_config = {"video": ....}

Reply:
Ok, i'll do it:
```
bg_config = {
    "video": get_background_config("video"),
    "audio": get_background_config("audio"),
}
```